### PR TITLE
network: add s390 options to default ifcfg files (#1074570)

### PR DESCRIPTION
**Pull request by rvykydal**
Resolves: rhbz#1074570

### CODE REVIEW

File: pyanaconda/network.py [hunk 1]
```diff
@@ -537,6 +538,18 @@ def add_connection_for_ksdata(networkdata, devname):
         values.append(['connection', 'id', devname, 's'])
         values.append(['connection', 'interface-name', devname, 's'])
 
+        # Add s390 settings
```
Comment (bcl) [inline pyanaconda/network.py]: Shouldn't this check to make sure it is running on s390?

File: pyanaconda/network.py [hunk 2]
```diff
@@ -547,6 +560,38 @@ def add_connection_for_ksdata(networkdata, devname):
...
+        return cfg
+    cfg['SUBCHANNELS'] = ','.join(subchannels)
+
+    ## cat /etc/ccw.conf
+    #qeth,0.0.0900,0.0.0901,0.0.0902,layer2=0,portname=FOOBAR,portno=0
+    #
+    #SUBCHANNELS="0.0.0900,0.0.0901,0.0.0902"
+    #NETTYPE="qeth"
+    #OPTIONS="layer2=1 portname=FOOBAR portno=0"
+    with open('/run/install/ccw.conf') as f:
```
Comment (bcl) [inline pyanaconda/network.py]: Should probably add a check for the file's existance, even though on non-s390 the subchannel check above will return the default dict and skip this.

**Comment by bcl** [comment]
Looks ok other than those comments.

File: pyanaconda/network.py (hunk 1 above)
Comment (rvykydal) [inline pyanaconda/network.py]: I thought it is not necessary but now I think we don't want to call the stuff on non-s390 arch, and we are already checking for s390 arch in the module, so I'll add the check.

File: pyanaconda/network.py (hunk 2 above)
Comment (rvykydal) [inline pyanaconda/network.py]: Okay, I'll add it.

**Comment by rvykydal** [comment]
Pushed with changes proposed by bcl.

**Pull request closed without merging**